WalletWasabi.Fluent/ViewModels/Wallets/Home/History/HistoryViewModel.cs

```diff
@@ -131,13 +129,12 @@ public HistoryViewModel(WalletViewModel walletViewModel, IObservable<Unit> updat
 						MinWidth = new GridLength(145, GridUnitType.Pixel),
 						MaxWidth = new GridLength(210, GridUnitType.Pixel)
 					},
-					width: new GridLength(0, GridUnitType.Auto),
-					numberOfPrivacyChars: 9),
+					width: new GridLength(0, GridUnitType.Auto)),
 
 				// Outgoing
-				new PrivacyTextColumn<HistoryItemViewModelBase>(
+				new TemplateColumn<HistoryItemViewModelBase>(
```

Comment:
The PrivacyTextColumn was specifically used to improve history table performance, especially scrolling perf.